Gopkg.toml

```diff
@@ -29,33 +29,33 @@
   name = "github.com/Sirupsen/logrus"
   version = "1.4.1"
 
-[[constraint]]
-  name = "github.com/prometheus/client_golang"
-  version = "0.9.2"
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.15.4"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.1"
+  version = "kubernetes-1.15.4"
 
 [[override]]
-  name = "sigs.k8s.io/controller-runtime"
-  version = "=v0.1.10"
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.15.4"
 
 [[override]]
-  name = "github.com/litmuschaos/chaos-operator"
-  branch = "master"
-  
+  name = "github.com/coreos/prometheus-operator"
```

Comment:
are we really using this somehere in executor?

Reply:
Removing it!